### PR TITLE
DOC: some proofreading

### PR DESCRIPTION
**Pull request by mrakitin**
Some minor edits/corrections of the docs.

### CODE REVIEW

File: docs/source/api.rst [hunk 1]
```diff
@@ -17,10 +17,10 @@ order) and random access (requesting a specific entry by name) by implementing
 ``__iter__`` and ``__getitem__`` respectively.
 
 It should also implement ``__contains__`` because, similarly, if
-``__contains__`` if contains specifically implemented, Python will iterate
-through all the entries and check each in turn. In this case, it is likely more
-efficient to implement a ``__contains__`` method that uses ``__getitem__`` to
-determine whether a given key in conatined.
+``__contains__`` specifically implemented, Python will iterate through all the
```
Comment (danielballan) [inline docs/source/api.rst]:
```suggestion
``__contains__`` is specifically implemented, Python will iterate through all the
```

File: docs/source/api.rst [hunk 2]
```diff
@@ -17,10 +17,10 @@ order) and random access (requesting a specific entry by name) by implementing
...
+entries and check each in turn. In this case, it is likely more efficient to
+implement a ``__contains__`` method that uses ``__getitem__`` to determine
+whether a given key in contained.
```
Comment (danielballan) [inline docs/source/api.rst]:
```suggestion
whether a given key is contained.
```

**Comment by mrakitin** [comment]
Thanks for the extra-suggestions, @danielballan, committed via the "Suggested edits" GH feature. I think it's good to go. Feel free to merge when ready.

**Comment by mrakitin** [comment]
Thank you.

**Pull request merged**